CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### Upcoming Changes
 
+* Add support for SquareRoot and U256SquareRoot under `cairo-1-hints` [#1104](https://github.com/lambdaclass/cairo-rs/pull/1104)
```

Comment:
I think it is not necessary to update the changelog as it only test.
Lets talk it tomorrow in the daily

Reply:
remove it